iroh-qrpc-client/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "iroh-qrpc-client"
+version = "0.2.0"
+edition = "2021"
+authors = ["Friedel Ziegelmayer <me@dignifiedquire.com>", "ramfox"]
+license = "Apache-2.0/MIT"
+repository = "https://github.com/n0-computer/iroh"
+description = "RPC type client for iroh"
+rust-version = "1.65"
+
+[dependencies]
+
+anyhow = "1"
```

Comment:
```suggestion
anyhow = { version = "1", features = ["backtrace"] }
```

Reply:
These are dead files that for some reason I missed.